### ISSUE TITLE
Pre-work complete for Android bootcamp

### ISSUE THREAD

**Issue by manishkantamneni** [manishkantamneni/helloworld-codepath#1]
Hello!

I've submitted my pre-work for Android bootcamp (ft. Uber). Please review it. /cc @codepathreview @codepath

**Comment by nesquena** [comment]
Thanks for submitting the Todo app. Can you please review the section in the pre-work about [the readme](http://courses.codepath.com/snippets/intro_to_android/prework#heading-4-submitting-the-project) for your project? In particular, check out this [README template](http://courses.codepath.com/snippets/intro_to_android/readme_templates/prework_readme.md?raw=true) and be sure to [include a GIF recorded with LiceCap](http://www.cockos.com/licecap/). Please ping me back here once that's complete.

**Comment by manishkantamneni** [comment]
Hi Nathan,

I did submit the GIF but in project directory and not in the readme file.
https://github.com/manishkantamneni/helloworld-codepath/blob/master/bootcamp-1.gif

Should I move it to the readme file?

On Fri, Jun 24, 2016 at 12:34 AM, Nathan Esquenazi <notifications@github.com

**Comment by nesquena** [comment]
Yes, we recommend you use this [README template](http://courses.codepath.com/snippets/intro_to_android/readme_templates/prework_readme.md?raw=true)
